resources/eventing/charts/nats/templates/configmap.yaml

```
@@ -155,3 +155,17 @@ data:
     {{- with .Values.nats.limits.lameDuckDuration }}
     lame_duck_duration: {{ . }}
     {{- end }}
+
+    {{- if .Values.auth.enabled }}
+    ##################
+    #                #
+    # Authorization  #
+    #                #
+    ##################
+    {{- if .Values.auth.resolver }}
+    {{- if eq .Values.auth.resolver.type "memory" }}
+    resolver: MEMORY
+    include "accounts/{{ .Values.auth.resolver.configMap.key }}"
```

```suggestion
    include "accounts/{{ .Values.auth.resolver.secret.key }}"
```

Related to: https://github.com/kyma-project/kyma/pull/15274/files#r959474476